src/components/Account-panel/Change-password/Change-password.tsx

```diff
@@ -15,19 +17,29 @@ type FormData = {
   new_password_confirm: string
 }
 
-const stInput = {
+export const stInput = {
   '& input': {
+    color: 'secondary.darker',
     backgroundColor: '#ffffff'
+  },
+  '&.Mui-disabled': {
+    color: 'secondary.darker'
+
+    // opacity: 0.8
   }
 }
 
-const stBtn = {
+export const stBtn = {
```

Comment:
remove this styles, we already have most of them for `Button`, use only you need insisde the component `sx` prop, not as a variable

Reply:
done